frontend/src/lib/components/PropertyFilters/utils.ts

```diff
@@ -26,3 +27,31 @@ export function isValidPropertyFilter(filter: AnyPropertyFilter): filter is Prop
         Object.values(filter).some((v) => !!v) // contains some properties with values
     )
 }
+
+export function filterMatchesItem(
+    filter?: AnyPropertyFilter | null,
+    item?: EventDefinition | null,
+    itemType?: string
+): boolean {
+    if (!filter || !item || !itemType || filter.type !== itemType) {
+        return false
+    }
+    return filter.type === 'cohort' ? filter.value === item.id : filter.key === item.name
+}
+
+const propertyFilterMapping: Record<string, TaxonomicFilterGroupType> = {
+    person: TaxonomicFilterGroupType.PersonProperties,
+    event: TaxonomicFilterGroupType.EventProperties,
+    cohort: TaxonomicFilterGroupType.Cohorts,
+    element: TaxonomicFilterGroupType.Elements,
+}
+
+export function propertyFilterTypeToTaxonomicFilterType(
+    filterType?: string | null
+): TaxonomicFilterGroupType | undefined {
+    return filterType && filterType in propertyFilterMapping ? propertyFilterMapping[filterType] : undefined
+}
+
+export function taxonomicFilterTypeToPropertyFilterType(filterType?: TaxonomicFilterGroupType): string | undefined {
```

Comment:
Minor stylistic question: Why have a function do this every time with find, rather than reverse the mapping of `propertyFilterMapping` once and just use object attributes directly? Wouldn't `Object.fromEntries(Object.entries(propertyFilterMapping).map(([ k, v ]) => [v, k]))` accomplish that?

Reply:
It would. Though this is running in `onChange` handlers only, so the performance boost will be pretty tiny. I'll fix it in a follow up PR. Hope to get this in its current state in front of as many eyeballs as quickly as possible.